.github/workflows/main.yml

```diff
@@ -1,63 +1,98 @@
 ---
 name: Run Tests
 
+# yamllint disable-line rule:truthy
 on:
   push:
-    branches: [main, "v*"]
+    branches:
+      - main
+      - v*
   pull_request:
-    branches: [main, "v*"]
+    branches:
+      - main
+      - v*
   release:
-    types: [released]
+    types:
+      - released
 
 jobs:
   lint:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: actions/setup-python@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.11'
       - uses: pre-commit/action@v3.0.0
         with:
           extra_args: -a
```

Comment:
we don't need this anymore I think 😊 we use pre-commit.ci

Reply:
Oh, that's nice to know! Will remove this then